pkg/agent/sysadvisor/plugin/inference/inference.go

```diff
@@ -68,14 +74,25 @@ func NewInferencePlugin(pluginName string, conf *config.Configuration, extraConf
 		return nil, fmt.Errorf("nil emitterPool")
 	}
 
-	emitter := emitterPool.GetDefaultMetricsEmitter().WithTags("advisor-inference")
+	metricEmitter := emitterPool.GetDefaultMetricsEmitter().WithTags("advisor-inference")
+
+	dataEmitter, err := emitterPool.GetMetricsEmitter(metricspool.PrometheusMetricOptions{
```

Comment:
it seems that dataEmitter isn't used in inferencePlugin?

Reply:
forgot to delete that.
removed already.